database/seeders/IdentitySeeder.php

```diff
@@ -4,221 +4,235 @@
-            [
-                'name' => __('Southeast Asian'),
-                'clusters' => ['ethnoracial'],
-            ],
-            [
-                'name' => __('Middle Eastern'),
-                'clusters' => ['ethnoracial'],
-            ],
-            [
-                'name' => __('Latin American'),
-                'clusters' => ['ethnoracial'],
-            ],
-            [
-                'name' => __('African'),
-                'clusters' => ['ethnoracial'],
-            ],
-            [
-                'name' => __('Women'),
-                'clusters' => ['gender', 'gender-and-sexuality'],
-            ],
-            [
-                'name' => __('Men'),
-                'clusters' => ['gender', 'gender-and-sexuality', 'reachable-when-mixed'],
-            ],
-            [
-                'name' => __('Non-binary people'),
-                'clusters' => ['gender', 'gender-and-sexuality', 'gender-diverse'],
-            ],
-            [
-                'name' => __('Gender non-conforming people'),
-                'clusters' => ['gender', 'gender-and-sexuality', 'gender-diverse'],
-            ],
-            [
-                'name' => __('Gender fluid people'),
-                'clusters' => ['gender', 'gender-and-sexuality', 'gender-diverse'],
-            ],
-            [
-                'name' => __('First Nations'),
-                'clusters' => ['indigenous'],
-            ],
-            [
-                'name' => __('Inuit'),
-                'clusters' => ['indigenous'],
-            ],
-            [
-                'name' => __('Métis'),
-                'clusters' => ['indigenous'],
-            ],
-            [
-                'name' => __('Supporters'),
-                'clusters' => ['lived-experience', 'reachable-when-mixed'],
-            ],
-        ];
+        // check if there is a JSON file that has stored data for the seeder
+        if (Storage::disk('seeds')->exists('identities.json')) {
+            $identities = json_decode(Storage::disk('seeds')->get('identities.json'), true);
 
-        foreach ($identities as $identity) {
-            Identity::firstOrCreate([
-                'name' => [
-                    'en' => $identity['name'],
-                    'fr' => trans($identity['name'], [], 'fr'),
-                ],
-                'description' => [
-                    'en' => $identity['description'] ?? null,
-                    'fr' => isset($identity['description']) ? trans($identity['description'], [], 'fr') : null,
-                ],
-                'clusters' => $identity['clusters'] ?? [],
-            ]);
+            foreach ($identities as $identity) {
+                Identity::firstOrCreate([
+                    'name' => json_decode($identity['name'], true),
+                    'description' => json_decode($identity['description'], true),
+                    'clusters' => json_decode($identity['clusters'], true),
+                ]);
```

Comment:
I'm not sure exactly but could this have been written as?
```suggestion
                Identity::firstOrCreate(json_decode($identity, true));
```

Reply:
No. `$identity` is already an object but the property values are strings that need decoded.